divtel/ArrayConfig/pointing.py

```diff
@@ -0,0 +1,127 @@
+"""
+Functions to define telescopes pointings
+We use the same reference frame as simtel_array:
+X is pointing North
+Y is pointing East
+Z is pointing upward
+Az is taken clock-wise from X (towards Y) and between 0 and 180 degrees
+Alt is taken from ground (towards Z) and between 0 and 90 degrees
+"""
+
+import numpy as np
+import astropy.units as u
+from astropy.coordinates import SkyCoord, ICRS
+
+
+def alt_az_to_vector(alt, az):
```

Comment:
Astropy coordinates directly support cartesian coordinates (also works for altaz)

```
In [1]: from astropy.coordinates import SkyCoord

In [2]: c = SkyCoord.from_name("Crab Nebula")

In [3]: c.cartesian
Out[3]: 
<CartesianRepresentation (x, y, z) [dimensionless]
    (0.10280914, 0.92137099, 0.37484102)>
```